framework/test/unit/modules/auth/command.spec.ts

```diff
@@ -0,0 +1,480 @@
+/*
```

Comment:
this file should be 
`framework/test/unit/modules/auth/commands/register_multisignature.spec.ts`?